### PR TITLE
Introduce optional `minimum_coverage_percentage` parameter which raises user error if overall coverage below specified threshold

### PR DESCRIPTION
**Pull request by opfeffer**
Proposing an additional, optional parameter `-m` which allows users to specify a minimum coverage percentage.

`xcov -w MyProject.xcworkspace -s MyProject -o xcov_output -m 75`

If the report returns an overall code coverage percentage below specified threshold a user error would be thrown and a non-zero exit code would be returned.

This allows enforcement of code coverage rules on various CI systems such as Jenkins.

### CODE REVIEW

File: lib/xcov/manager.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class Manager
     def work(options)
       Xcov.config = options
       FastlaneCore::PrintTable.print_values(config: options, hide_keys: [:slack_url], title: "Summary for xCov #{Xcov::VERSION}")
-      Runner.new.run
+      return Runner.new.run
```
Comment (nakiostudio) [inline lib/xcov/manager.rb]: No real need of changing it!

**Comment by nakiostudio** [comment]
Hi @opfeffer, amazing feature, thanks for your contribution :). Just update the PR with my silly comment, squash the commits and we are ready to go!

File: lib/xcov/manager.rb (hunk 1 above)
Comment (opfeffer) [inline lib/xcov/manager.rb]: Ah yes, sorry -- somewhat a ruby beginner :+1: Removing

**Comment by opfeffer** [comment]
Thanks! Question, should i bump the .gemspec also or are you taking care of this when pulling in my changes?

**Comment by nakiostudio** [comment]
@opfeffer I will take care of it :+1:

**Comment by opfeffer** [comment]
Done! Thanks much for your feedback!

**Comment by nakiostudio** [comment]
No problem, thanks for the PR :smiley: I will release 0.3 later today :+1:

**Pull request merged**